DOCS/man/mpv.rst

```diff
@@ -371,6 +371,79 @@ To use this feature, you need to fill the ``menu-data`` property with menu
 definition data, and add a keybinding to run the ``context-menu`` command,
 which can be done with a user script.
 
+Image Bindings
+--------------
+
+Several key bindings are changed when viewing images:
+
+LEFT, RIGHT, UP, DOWN, h, j, k, l
+    Pan the image when it is larger than the window.
+
+Ctrl+LEFT, Ctrl+RIGHT, Ctrl+UP, Ctrl+DOWN, Ctrl+h, Ctrl+j, Ctrl+k, Ctrl+l
+    Align the image to one of the boundaries of the window when it is larger
+    than the window.
+
+= and -
+    Change the zoom.
+
++ and _
+    Change the zoom slowly.
+
+0
+    Reset the zoom.
+
+u
+    Toggle between showing the image unscaled in its original dimensions and
+    fitting it to the window.
+
+o
+    Fill the window with the image.
+
+s
+    Toggle between showing images for 5 seconds in a slideshow and keeping them
+    open forever.
+
+n
+    Go to the next playlist entry.
+
+p
+    Go to the previous playlist entry.
+
+N
+    Go to the next sub-playlist (e.g. directory or achive).
+
+P
+    Go to the previous sub-playlist.
+
+g-g
```

Comment:
I see what you did here.

Reply:
We can also bind HOME and END if desired.

Reply:
Is this really image specific though?

Reply:
No, but it is more useful for images than for videos. I also use it for videos and audio and would add it to the regular input.conf, but dunno if that's wanted, up to you. We do have G bound to add sub-scale unfortunately (I bind - and = for that).